src/v/kafka/server/rm_group_frontend.cc

```diff
@@ -37,13 +37,21 @@ using namespace std::chrono_literals;
  */
 ss::future<bool> try_create_consumer_group_topic(
   kafka::coordinator_ntp_mapper& mapper,
-  cluster::topics_frontend& topics_frontend) {
+  cluster::topics_frontend& topics_frontend,
+  int16_t node_count) {
+    // Attempt to use internal topic replication factor, if enough nodes found.
+    auto replication_factor
+      = (int16_t)config::shard_local_cfg().internal_topic_replication_factor();
+    if (replication_factor > node_count) {
+        replication_factor = 1;
```

Comment:
Why it is not better to use `node_count` as `replication_factor` here?

Reply:
If a cluster is in the process of starting, then I don't want to try and create a replicas=2 partition